lib/token.js

```diff
@@ -271,22 +271,26 @@ function convertOAuthParamsToQueryParams(oauthParams) {
   // Convert our params to their actual OAuth equivalents
   var oauthQueryParams = util.removeNils({
```

Comment:
This list was getting long, and to make it easier to compare to the docs I sorted it.  Unfortunately this means updating all the unit tests that specified the expected order.  

Params are now in alphabetical order except for scope, which is always last.